src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
+
+  if (days >= 7) {
+    return rentalCost - 50;
```

Comment:
Replace the literal '7' with a named constant such as 'LONG_TERM_RENTAL_THRESHOLD' to clearly indicate its purpose.

Reply:
not fixed